vm/src/obj/objint.rs

```diff
@@ -398,10 +398,12 @@ impl PyInt {
     }
 
     #[pymethod(name = "__hash__")]
-    fn hash(&self, _vm: &VirtualMachine) -> u64 {
```

Comment:
I'm unsure whether we must use `u64` or `usize` for hashes. What was your reason to switch from `u64` to `usize`?

Reply:
Just to unify to other __hash__ impls, because some of them returns u64 and the others returns usize for now.

Switching to `isize` is also an option because it is Py_ssize_t in CPython.

Reply:
#927 met a problem related to hash size. If we can't decide the type at this time, I suggest to introduce a new type alias `type HashValue = u64` (or usize) and keep watching how it goes.

Reply:
@windelbouwman I fixed type definition based on CPython. Will you check it again?